internal/pkg/blockchain-scrapers/blockchains/ethereum/scrapers/eth/main.go

```diff
@@ -42,7 +42,7 @@ func init() {
 	premine.SetString("72009990499480000000000000", 10)
 	//Checkpoints - Hardcoded - Starting point, even if DB/persistant layer is unavailable
 	//Might be good idea to update these while performing mantainence on this code.
-	checkpointBlock = big.NewInt(6407102 )
+	checkpointBlock = big.NewInt(6407102)
 	checkpointSupply.SetString("30202142593750000000000000", 10)
 	// Override the above from some database
```

Comment:
why changing this file ?

Reply:
gofmt changed it. As well as all the other files

Reply:
then remove it from this PR